robot_sf/benchmark/metrics.py

```
@@ -2592,7 +2602,11 @@ def compute_all_metrics(  # noqa: PLR0913
         and total_collision_count == 0
     )
 
-    values: dict[str, float] = {}
+    values: dict[str, Any] = {}
+    try:
+        values.update(calculate_signal_metrics(data))
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)